python/hail/context.py

```diff
@@ -114,11 +114,43 @@ def init(sc=None, app_name="Hail", master=None, local='local[*]',
              log='hail.log', quiet=False, append=False,
              min_block_size=1, branching_factor=50, tmp_dir='/tmp',
              default_reference="GRCh37"):
-    HailContext(sc, app_name, master, local, log, quiet, append, min_block_size, branching_factor, tmp_dir, default_reference)
+    """Initialize Hail and Spark.
+
+    Parameters
+    ----------
+    sc : pyspark.SparkContext, optional
+        Spark context. By default, a Spark context will be created.
+    app_name : :obj:`str`
+        Spark application name.
+    master : :obj:`str`
```

Comment:
If optional means nullable, then `master` should also be optional, with a description of the default behavior. I see that the null case is explained in the description of `local`, but I think it should be repeated here.